SCons/SConf.py

```diff
@@ -733,9 +736,6 @@ def _createDir( self, node ):
             if not os.path.isdir( dirName ):
                 os.makedirs( dirName )
 
-    def _set_conftest_node(self, node):
```

Comment:
no reason for this to be part of the class and muck around with self arg, moved it to an isolated function

Reply:
Or @staticmethod